docs/guides/private-submodules.rst

```diff
@@ -1,92 +1,106 @@
-Using Private Git Submodules with Read the Docs for Business
-============================================================
+Using Private Git Submodules
+============================
+
+.. warning::
+
+   This guide is for :doc:`/commercial/index`.
 
 Read the Docs uses SSH keys (with read only permissions) in order to clone private repositories.
-An SSH key is automatically generated and added to your main repository, but not on your submodules.
+A SSH key is automatically generated and added to your main repository, but not to your submodules.
 In order to give Read the Docs access to clone your submodules you'll need to add the public SSH key to each repository of your submodules.
 
 .. note::
 
    You can manage which submodules Read the Docs should clone using a configuration file.
    See :ref:`config-file/v2:submodules`.
 
-Project's SSH Key
```

Comment:
You can add like a small table of contents before this title so when reading the first section you will immediately realize that there are just 2 steps, but the second depends on your VCS provider --without reading the whole doc, and you can jump to the section easily.

Similar to what we are doing at: https://docs.readthedocs.io/en/latest/api/v3.html

Reply:
I just added the TOC, looks like this is already shown in the sidebar